lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -26,103 +28,124 @@ class ExpansionContactListTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.only(left: 8.0, top: 8.0, bottom: 12.0),
-      child: FutureBuilder<Profile?>(
-        key: contact.matrixId != null ? Key(contact.matrixId!) : null,
-        future:
-            contact.status == ContactStatus.active ? getProfile(context) : null,
-        builder: (context, snapshot) {
-          return Row(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              IgnorePointer(
-                child: Avatar(
-                  mxContent: snapshot.data?.avatarUrl,
-                  name: contact.displayName,
-                  size: 48.0,
+    return TwakeListItem(
+      child: Padding(
+        padding: const EdgeInsets.only(left: 8.0, top: 8.0, bottom: 8.0),
+        child: FutureBuilder<Profile?>(
+          key: contact.matrixId != null ? Key(contact.matrixId!) : null,
+          future: contact.status == ContactStatus.active
+              ? getProfile(context)
+              : null,
+          builder: (context, snapshot) {
+            return Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
```

Comment:
`CrossAxisAlignment.center` is the default value so we don't need to set it again.